torch/_dynamo/variables/lists.py

```diff
@@ -15,12 +15,35 @@
 from ..exc import unimplemented
 from ..guards import make_dupe_guard
 from ..source import GetItemSource
-from ..utils import check_constant_args, get_fake_value, guard_if_dyn, namedtuple_fields
+from ..utils import get_fake_value, guard_if_dyn, namedtuple_fields
 from .base import MutableLocal, VariableTracker
 from .constant import ConstantVariable
 from .functions import UserFunctionVariable, UserMethodVariable
 
 
+def _listlike_contains_helper(items, search, options):
+    if search.is_python_constant():
+        result = any(
+            x.as_python_constant() == search.as_python_constant() for x in items
+        )
+        return variables.ConstantVariable(result, **options)
+
+    from .builtin import BuiltinVariable
+
+    result = None
+    for x in items:
+        check = BuiltinVariable(operator.eq).call_function(tx, [x, search], {})
+        if result is None:
+            result = check
+        else:
+            result = BuiltinVariable(operator.or_).call_function(
+                tx, [check, result], {}
+            )
+    if result is None:
```

Comment:
I noticed this will run over every item, but contains can stop on the first one, I think? I know this is the same code as before, but I wonder if we can make it return early?

Reply:
Unfortunately we can't, as we have to traverse the whole list/set to get the result. E.g, ```False, False, False, True```, we can't have the result before we hit the last element.

Reply:
ack